Makefile

```diff
@@ -47,7 +47,9 @@ jenkins: requirements ## Run Jenkins playbook
 	JENKINS_VARS_FILE=$$(mktemp) ;\
 	PRIVATE_KEY_FILE=$$(mktemp) ;\
 	trap 'rm $$JENKINS_VARS_FILE $$PRIVATE_KEY_FILE' EXIT ;\
-	${DM_CREDENTIALS_REPO}/sops-wrapper -d ${DM_CREDENTIALS_REPO}/jenkins-vars/jenkins.yaml > $$JENKINS_VARS_FILE ;\
+	for varfile in ${DM_CREDENTIALS_REPO}/jenkins-vars/*.yaml; do \
```

Comment:
Should we allow `.yml` as well as `yaml` for flexibility, or should we enforce consistency? (I am on the fence...)

Reply:
Google says... https://stackoverflow.com/questions/21059124/is-it-yaml-or-yml?rq=1

Reply:
I prefer consistency

Reply:
Also the only yaml files in the relevant folder so far are .yaml so hopefully that means that we shouldn't need the flexibility

Reply:
Sounds good to me!